src/NovelAudioService.cpp

```diff
@@ -331,6 +316,34 @@ std::string NovelAudioService::findByChannelMap(int channel) {
   return nullptr;
 }
 
+void NovelAudioService::logIfSDLFailure(int (*function)(Uint32), Uint32 sdl_flag) {
```

Comment:
Is there a reason were taking a raw function pointer here and not an `std::function<T>`?

Reply:
There was an issue during original implementation, however it was just a knowledge gap on my end. I've updated the code and this will be resolved next commit - no more raw function pointers 😃